Src/Base/AMReX_TinyProfiler.H

```diff
@@ -24,12 +24,21 @@ class TinyProfiler
 public:
     explicit TinyProfiler (std::string funcname) noexcept;
     TinyProfiler (std::string funcname, bool start_) noexcept;
+#ifdef AMREX_USE_CUPTI
+    TinyProfiler (std::string funcname, bool start_, bool useCUPTI) noexcept;
+#endif // AMREX_USE_CUPTI
     explicit TinyProfiler (const char* funcname) noexcept;
     TinyProfiler (const char* funcname, bool start_) noexcept;
+#ifdef AMREX_USE_CUPTI
+    TinyProfiler (const char* funcname, bool start_, bool useCUPTI) noexcept;
+#endif // AMREX_USE_CUPTI
     ~TinyProfiler ();
```

Comment:
Because the CUPTI macros are separate macros, there's no need for an additional function. Can just add a defaulted (useCUPTI = false) to the current ones.

Reply:
Modified so that 'useCUPTI = false' is the default.